ui/components/app/detected-token/detected-token-selection-popover/detected-token-selection-popover.js

```diff
@@ -34,10 +37,31 @@ const DetectedTokenSelectionPopover = ({
   const chainId = useSelector(getCurrentChainId);
 
   const detectedTokens = useSelector(getDetectedTokensInCurrentNetwork);
+  const { tokenNetworkFilter } = useSelector(getPreferences);
+
+  const currentNetwork = useSelector(getCurrentNetwork);
+
+  const detectedTokensMultichain = useSelector(
+    getAllDetectedTokensForSelectedAddress,
+  );
+
+  const totalTokens = useMemo(() => {
+    return process.env.PORTFOLIO_VIEW &&
+      Object.keys(tokenNetworkFilter).length > 1
+      ? Object.values(detectedTokensMultichain).reduce(
+          (count, tokenArray) => count + tokenArray.length,
+          0,
+        )
+      : detectedTokens.length;
+  }, [detectedTokensMultichain, detectedTokens, tokenNetworkFilter]);
```

Comment:
Can this also use my selector? https://github.com/MetaMask/metamask-extension/pull/28502